packages/blockchain-api/tslint.json

```diff
@@ -1,3 +1,6 @@
 {
-  "extends": ["@celo/typescript/tslint.json"]
+  "extends": ["@celo/typescript/tslint.json"],
+  "rules": {
+    "no-floating-promises": true
```

Comment:
Can you add this to the root tslint config instead of adding it to projects specifically?

Reply:
> Can you add this to the root tslint config instead of adding it to projects specifically?

1. I don't see a root one
2. I cannot enable this on all packages yet since many, especially mobile, have several lint failures.

Reply:
The root one is stated on line 2: @celo/typescript/tslint.json"